### ISSUE TITLE
Get "Failed to verify the source. Please check the address." under latest docker version

### ISSUE THREAD

**Issue by Jacob1532** [datarhei/restreamer#455]
**Describe the bug**
I have running restreamer since more then 6 months now and no problems.
I update the recreate the docker container today and couldn't establish the connection to the RTSP-Stream. I recreate the container in regular terms, and normally I don't even have to re establish the connection. But today I received the error "Failed to verify the source. Please check the adress.
I check the adress "rtsp://X.X.X.15:8554/stream" via VLC the RTSP stream is accessable.
I just jump back to version "rpi-2.3.0" and there everything is running smoothly again.

**To Reproduce**
Under both versions:
1. Add new source --> network source
2. enter Adress "rtsp://X.X.X.15:8554/stream"

It runs with the previous version, without any issues:
sudo docker run -d --restart=always --name restreamer -v /opt/restreamer/config:/srv/restreamer/config -v /opt/restreamer/data:/srv/restreamer/data -p 8080:8080 -p 8181:8181 -p 1935:1935 -p 1936:1936 -p 6000:6000/udp datarhei/restreamer:rpi-2.3.0
and I can set up the RTSP stream.

Upgrade the docker container with the latest version:
sudo docker run -d --restart=always --name restreamer_test -v /opt/restreamer/config:/srv/restreamer/config -v /opt/restreamer/data:/srv/restreamer/data -p 8080:8080 -p 8181:8181 -p 1935:1935 -p 1936:1936 -p 6000:6000/udp datarhei/restreamer:latest

I get the error message: "Failed to verify the source. Please check the address." when entering the RTSP-adress.

**Expected behavior**
The latest version should also recognize the rtsp stream.

**Server:**
It running in on a 
RETTY_NAME="Debian GNU/Linux 10 (buster)"
NAME="Debian GNU/Linux"
VERSION_ID="10"
VERSION="10 (buster)"
VERSION_CODENAME=buster
ID=debian
with docker version 
Docker version 20.10.21, build baeda1f

Thanks,
Jacob

**Comment by jstabenow** [comment]
Error detected ;-) 

```json
{
   "streams":[],
   "log":[
      "ffmpeg version 5.1.2-datarhei Copyright (c) 2000-2022 the FFmpeg developers",
      "  built with gcc 11.2.1 (Alpine 11.2.1_git20220219) 20220219",
      "  configuration: --extra-version=datarhei --prefix=/usr --extra-libs='-lpthread -lxml2 -lm -lz -lsupc++ -lstdc++ -lssl -lcrypto -lz -lc -ldl' --enable-nonfree --enable-gpl --enable-version3 --enable-postproc --enable-static --enable-openssl --enable-libxml2 --enable-libv4l2 --enable-v4l2_m2m --enable-libfreetype --enable-libsrt --enable-libx264 --enable-libx265 --enable-libvpx --enable-libmp3lame --enable-libopus --enable-libvorbis --disable-ffplay --disable-debug --disable-doc --disable-shared",
      "  libavutil      57. 28.100 / 57. 28.100",
      "  libavcodec     59. 37.100 / 59. 37.100",
      "  libavformat    59. 27.100 / 59. 27.100",
      "  libavdevice    59.  7.100 / 59.  7.100",
      "  libavfilter     8. 44.100 /  8. 44.100",
      "  libswscale      6.  7.100 /  6.  7.100",
      "  libswresample   4.  7.100 /  4.  7.100",
      "  libpostproc    56.  6.100 / 56.  6.100",
      "Unrecognized option 'stimeout'.",
      "Error splitting the argument list: Option not found"
   ]
}
```

Damn. Will fix it asap. Thx for your feedback!

**Comment by svenerbeck** [comment]
Dear @Jacob1532 
We are closing your issue #455.

This may be due to the following reasons:

- Problem/inquiry has been solved (Version 2.4.1)
- Ticket remained unanswered by you for a longer period
- Problem was explained and handled in another ticket

You can reopen this ticket at any time!

Please only open related tickets once! Always answer/ask in the original ticket with the same issue!

💛 If you were satisfied with the support and we could help you, please **[rate us on Google](https://g.page/r/CSv68cfIr9l9EBE/review)**. 

With kind regards,
Your datarhei team
